docs/core/INPUT.md

```diff
@@ -0,0 +1,60 @@
+# Input
+
+## Input System
+In Bebone you can use event-based input system to add some actions to your keys. Input class — is a singleton class, so you can get it's instance anywhere in your codebase.
+
+Input system works with ```Action<>``` delegate. Because of this feature, you will subscribe/unsubscribe your keyboard keys and mouse buttons to the 'in-game actions' (such as jumping, running, etc.). You can do this with these two methods:
+```c++
+void register_key_action(const KeyCode& keyCode, std::function<void()>& action, const InputType& inputType = InputType::PRESS);
+void remove_key_action(const KeyCode& keyCode, std::function<void()>& action, const InputType& inputType = InputType::PRESS);
+```
+* KeyCode — enum, which specifies the key to which you can subscribe an action
+* action — 'in-game action'
+* inputType — enum, which specifies in which case the action will be executed (by default it is set to PRESS)
+
+All that is left to do is to add one command line inside the window loop to allow the framework to handle all the user input (maybe we will change it later):
```

Comment:
Change this text to:

Add this command line inside the window loop to allow the framework to handle all the user input (it will be removed in the future versions of **bebone**):